src/main.rs

```diff
@@ -70,6 +70,7 @@ const SHELL_OPTION: &str = "shell";
 const TASKS_OPTION: &str = "tasks";
 const FORCE_OPTION: &str = "force";
 const OUTPUT_DIR_OPTION: &str = "output-dir";
+const FORCE_IMAGE_PULL: &str = "force-image-pull";
```

Comment:
Can we move this to be right after the `FORCE_OPTION`? Then the "force" options would be together, which seems nice to me.